OsmAnd/src/net/osmand/plus/views/OsmandMapTileView.java

```diff
@@ -125,6 +127,7 @@ public class OsmandMapTileView implements IMapDownloaderCallback {
 
 	private CanvasColors canvasColors;
 	private Boolean nightMode;
+	private QuadPoint cachedAACanvasOffset = new QuadPoint();
```

Comment:
Does [displayPositionManager.setCustomMapRatio()](https://github.com/osmandapp/Osmand/blob/master/OsmAnd/src/net/osmand/plus/auto/SurfaceRenderer.java#L148) cannot be used for this?